src/org/linphone/setup/GenericLoginFragment.java

```diff
@@ -138,10 +152,94 @@ public void onClick(View v) {
 					}
 				}
 			});
-
+		sharedPreferences = PreferenceManager.getDefaultSharedPreferences(getActivity());
+		retrieveProviderTask = new ProviderNetworkOperation();
+		retrieveProviderTask.execute();
 		return view;
 	}
 
+	protected void srvLookup(String query){
+		try {
+			Record[] records = new Lookup(query, Type.SRV).run();
+			if (records != null) {
+				for (Record record : records) {
+					SRVRecord srv = (SRVRecord) record;
+
+					String hostname = srv.getTarget().toString().replaceFirst("\\.$", "");
+					int port = srv.getPort();
+					System.out.println(hostname + ":" + port);
```

Comment:
This is a debugging clue for testing SRV lookup of Config and in the future registration service. Need to comment the code